packages/lambdapi/lambdapi.2.1.0/opam

```diff
@@ -48,13 +48,12 @@ depends: [
   "sedlex" {>= "2.2"}
   "alcotest" {with-test}
   "alt-ergo" {with-test}
-  "alt-ergo" {<= "2.4.0"}
```

Comment:
what’s the reason for removing this dependency?

Reply:
The new release of Why3 is now compatible with Alt-ergo 2.4.1.

Reply:
Then the fix is to remove the constraint, not the dependency entirely

Reply:
Lambdapi does not actually depends on Alt-ergo. Alt-ergo is only used for a few tests (hence the dependency on line 50) and tests are not included in the build of Lambdapi.